app/models/reports/feedback_report.rb

```
@@ -0,0 +1,86 @@
+      teams.flat_map do |team|
+        team.users.filter_map do |user|
+          AssignmentParticipant.find_by(parent_id: @reportable.id, user_id: user.id)
+        end
+      end
+    end
+
+    def format_participant(p)
+      return {} unless p
+
+      {
+        id:        p.id,
```

RuboCop (1.86.2)

[convention] Align the keys of a hash literal if they span more than one line.

(Layout/HashAlignment)